core/src/main/java/hudson/model/Item.java

```diff
@@ -139,12 +139,11 @@
      *
      * @param g
      *      The {@link ItemGroup} instance used as context to evaluate the relative name of this item
-     * @return
-     *      The name of the current item, relative to p. Nested {@link ItemGroup}s are separated by {@code /} character.
      * @since 1.419
```

Comment:
If we keep the second `@return`, could we put this `@since` after it? just for readability when browsing the code.

Reply:
I don't understand how that's more readable. But I need another build anyway, so sure.